ISM43362/ISM43362.cpp

```diff
@@ -186,8 +186,12 @@ bool ISM43362::connect(const char *ap, const char *passPhrase)
     if (!(_parser.send("C2=%s", passPhrase) && check_response())) {
         return false;
     }
-    /* TODO security level = 3 , is it hardcoded or not ???? */
-    if (!(_parser.send("C3=3") && check_response())) {
+
+    /* Check security level is acceptable */
+    if (ap_sec > 5) {
```

Comment:
I'd rather define and use an enum that contains the values that are defined by the WIFI module spec.
For now ap_sec is equal to nsapi_security_t security;
but how can we be sure that nsapi_security_t  defines are in line with mdoules defines ?

Reply:
Thank you @LMESTM for the review.
You're right, we cannot assume that. 
Security > 4 are already diverging between mbed-os nsapi_security_t and ism43362.

I am now using a specific enum value, and return a debug printf if the security level is not correct.
Kind regards